gum/arch-arm64/gumarm64writer.c

```diff
@@ -561,24 +540,23 @@ gum_arm64_writer_put_push_reg_reg (GumArm64Writer * self,
                                    arm64_reg reg_a,
                                    arm64_reg reg_b)
 {
-  GumArm64RegInfo ra, rb, sp;
+    GumArm64RegInfo ra, rb, sp;
 
-  gum_arm64_writer_describe_reg (self, reg_a, &ra);
-  gum_arm64_writer_describe_reg (self, reg_b, &rb);
-  gum_arm64_writer_describe_reg (self, ARM64_REG_SP, &sp);
+    gum_arm64_writer_describe_reg(self, reg_a, &ra);
+    gum_arm64_writer_describe_reg(self, reg_b, &rb);
+    gum_arm64_writer_describe_reg(self, ARM64_REG_SP, &sp);
```

Comment:
Space between function name and opening parenthesis removed. Same regression below.